compiler/src/codegen/compcore.re

```diff
@@ -3518,6 +3518,14 @@ let compile_wasm_module = (~env=?, ~name=?, prog) => {
     compile_all();
   };
 
+  if (compiling_wasi_polyfill(name)) {
+    write_universal_exports(
```

Comment:
This is something that the linker already did, but it's needed to export plain wasm functions. There's no compiler flag for this because it would break linking (since Grain modules expect to import Grain values—if it helps you think about it, a module with universal exports could only be imported from via the `foreign wasm` syntax), and fully-linked modules export plain functions.